mm/iob/iob_clone.c

```diff
@@ -109,8 +109,8 @@ static int iob_next(FAR struct iob_s *iob, bool throttled, bool block)
  ****************************************************************************/
 
 int iob_clone_partial(FAR struct iob_s *iob1, unsigned int len,
```

Comment:
Why `unsigned` is removed and what is the reason for switching to subtraction logic?

Reply:
Because with signed offset, we may clone data before iob->io_offset, which contains our L2 data. Using subtraction logic is to make sure the compare result is correct (compare int and unsigned int may be risky since int will be cast to unsigned)